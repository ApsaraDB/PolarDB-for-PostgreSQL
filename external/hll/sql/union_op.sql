-- ----------------------------------------------------------------
-- Regression tests for union operator.
-- ----------------------------------------------------------------

SELECT hll_set_output_version(1);

SELECT hll_empty(11,5,256,1) || hll_empty(11,5,256,1);

SELECT hll_empty(11,5,256,1) || E'\\x128b498895a3f5af28cafe'::hll;

SELECT E'\\x128b498895a3f5af28cafe'::hll || E'\\x128b498895a3f5af28cafe'::hll;

SELECT E'\\x128b498895a3f5af28cafe'::hll || E'\\x128b49da0ce907e4355b60'::hll;

SELECT hll_empty(11,5,256,1) || hll_hash_integer(1,0);

SELECT hll_empty(11,5,256,1)
       || hll_hash_integer(1,0)
       || hll_hash_integer(2,0)
       || hll_hash_integer(3,0);

SELECT hll_hash_integer(1,0) || hll_empty(11,5,256,1);

SELECT hll_hash_integer(1,0) || hll_empty(11,5,256,1) || hll_hash_integer(2,0);
