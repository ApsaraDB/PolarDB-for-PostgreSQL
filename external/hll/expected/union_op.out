-- ----------------------------------------------------------------
-- Regression tests for union operator.
-- ----------------------------------------------------------------
SELECT hll_set_output_version(1);
 hll_set_output_version 
------------------------
                      1
(1 row)

SELECT hll_empty(11,5,256,1) || hll_empty(11,5,256,1);
 ?column? 
----------
 \x118b49
(1 row)

SELECT hll_empty(11,5,256,1) || E'\\x128b498895a3f5af28cafe'::hll;
         ?column?         
--------------------------
 \x128b498895a3f5af28cafe
(1 row)

SELECT E'\\x128b498895a3f5af28cafe'::hll || E'\\x128b498895a3f5af28cafe'::hll;
         ?column?         
--------------------------
 \x128b498895a3f5af28cafe
(1 row)

SELECT E'\\x128b498895a3f5af28cafe'::hll || E'\\x128b49da0ce907e4355b60'::hll;
                 ?column?                 
------------------------------------------
 \x128b498895a3f5af28cafeda0ce907e4355b60
(1 row)

SELECT hll_empty(11,5,256,1) || hll_hash_integer(1,0);
         ?column?         
--------------------------
 \x128b498895a3f5af28cafe
(1 row)

SELECT hll_empty(11,5,256,1)
       || hll_hash_integer(1,0)
       || hll_hash_integer(2,0)
       || hll_hash_integer(3,0);
                         ?column?                         
----------------------------------------------------------
 \x128b498895a3f5af28cafeda0ce907e4355b604848de7f7bd2a13b
(1 row)

SELECT hll_hash_integer(1,0) || hll_empty(11,5,256,1);
         ?column?         
--------------------------
 \x128b498895a3f5af28cafe
(1 row)

SELECT hll_hash_integer(1,0) || hll_empty(11,5,256,1) || hll_hash_integer(2,0);
                 ?column?                 
------------------------------------------
 \x128b498895a3f5af28cafeda0ce907e4355b60
(1 row)

